### PR TITLE
Split up existing data setup script

### PR DESCRIPTION
**Pull request by Gosifo**
## What?

Please include a summary of the change.

## Why?

Please include reason for the change and any other relevant context.

## Related PRs

Please include links to PRs in other repositories relevant to this PR.
Delete this section if not needed.

### CODE REVIEW

File: scripts/reset-test-users.sh [hunk 1]
```diff
@@ -2,27 +2,34 @@
 
 function resetTestUsers() {
 
+source ./scripts/reset-account-intervention.sh
```
Comment (CraigRyder) [inline scripts/reset-test-users.sh:5]: think this should be in 'reset-test-data.sh' at the top with the other two

**Review by CraigRyder — CHANGES_REQUESTED** [review]

File: scripts/reset-test-users.sh [hunk 2]
```diff
@@ -132,113 +139,122 @@ function resetTestUsers() {
...
-    $TEST_USER_PASSWORD \
-    $TEST_USER_PHONE_NUMBER \
-    $TEST_USER_LATEST_TERMS_AND_CONDITIONS_VERSION
 
-  createSMSUser \
-    $PERMANENTLY_LOCKED_ACCOUNT_EMAIL_1 \
-    $TEST_USER_PASSWORD \
-    $TEST_USER_PHONE_NUMBER \
-    $TEST_USER_LATEST_TERMS_AND_CONDITIONS_VERSION
+#  createSMSUser \
```
[the remark this answers is not in the document]
Comment (Gosifo) [reply scripts/reset-test-users.sh]: done

File: scripts/reset-test-users.sh [hunk 3]
```diff
@@ -2,27 +2,34 @@
...
-  deleteIntervention $PERMANENTLY_LOCKED_ACCOUNT_EMAIL_6
-  deleteIntervention $PERMANENTLY_LOCKED_ACCOUNT_EMAIL_7
-  deleteIntervention $TEST_USER_PASSWORD_RESET_ACCOUNT_EMAIL_1
-  deleteIntervention $TEST_USER_PASSWORD_RESET_ACCOUNT_EMAIL_2
-  deleteIntervention $TEST_USER_PASSWORD_RESET_ACCOUNT_EMAIL_4
-  deleteIntervention $TEST_USER_PASSWORD_RESET_ACCOUNT_EMAIL_5
-  deleteIntervention $TEST_USER_PASSWORD_RESET_ACCOUNT_EMAIL_6
-  deleteIntervention $TEST_USER_PASSWORD_RESET_ACCOUNT_EMAIL_7
-  deleteIntervention $ACCOUNT_INTERVENTION_REMOVED_EMAIL_1
+#  deleteIntervention $TEMPORARILY_SUSPENDED_ACCOUNT_EMAIL_1
```
[the remark this answers is not in the document]
Comment (Gosifo) [reply scripts/reset-test-users.sh]: done

File: scripts/reset-test-users.sh (hunk 1 above)
Comment (Gosifo) [reply scripts/reset-test-users.sh]: done

File: reset-test-data.sh [hunk 4]
```diff
@@ -42,3 +42,4 @@ if [ "${LOCAL}" == "1" ]; then
 fi
 
 resetTestUsers
+resetAccountIntervention
```
[the remark this answers is not in the document]
Comment (Gosifo) [reply reset-test-data.sh]: done

**Review by CraigRyder — APPROVED** [review]

**Pull request merged**